
 If you have an old version of PostgreSQL, for example,
 version 7.3 or older, it may not properly build with the current
 Bacula release due to incompatible changes in the PostgreSQL
 header files between version.  Only in the case that build fails,
 you might try applying this patch with:

 cd <bacula-2.2.7-source>
 patch -p1 <2.2.7-old-postgresql.patch
 ./configure <your-options>
 make
 ...
 make install


diff -uNr bacula-2.2.7/src/cats/postgresql.c bacula-2.2.7-fixed/src/cats/postgresql.c
--- bacula-2.2.7/src/cats/postgresql.c	2007-12-08 04:54:55.000000000 -0500
+++ bacula-2.2.7-fixed/src/cats/postgresql.c	2007-12-29 08:34:10.000000000 -0500
@@ -47,7 +47,6 @@
 #ifdef HAVE_POSTGRESQL
 
 #include "postgres_ext.h"       /* needed for NAMEDATALEN */
-#include "pg_config_manual.h"   /* get NAMEDATALEN on version 8.3 or later */
 
 /* -----------------------------------------------------------------------
  *
